xdsl/irdl/attributes.py

```diff
@@ -256,8 +262,30 @@ def irdl_attr_definition(cls: TypeAttributeInvT) -> TypeAttributeInvT:
     )
 
 
+IRDLGenericAttributeConstraint: TypeAlias = (
```

Comment:
```suggestion
IRDLGenericAttrConstraint: TypeAlias = (
```
To align with `GenericAttrConstraint`?